lib/vmdb/settings.rb

```diff
@@ -21,6 +22,11 @@ def initialize(errors)
     PASSWORD_FIELDS = Vmdb::SettingsWalker::PASSWORD_FIELDS
     DUMP_LOG_FILE   = Rails.root.join("log/last_settings.txt").freeze
 
+    # RESET_COMMAND remove record for selected key and specific resource
+    # RESET_ALL_COMMAND remove all records for selected key and all resources
+    MAGIC_VALUES = [RESET_COMMAND      = "<<reset>>".freeze,
+                    RESET_ALL_COMMAND  = "<<reset_all>>".freeze].freeze
```

Comment:
The definitions inside other definitions is weird, but... ![](https://i.imgur.com/sqtYh99.gif)